rotkehlchen/api/v1/schemas.py

```diff
@@ -3531,3 +3532,55 @@ class FalsePositveSpamTokenSchema(Schema):
 
 class SpamTokenSchema(Schema):
     token = AssetField(expected_type=EvmToken, required=True)
+
+
+class CalendarCommonEntrySchema(Schema):
+    name = fields.String(required=True)
+    description = fields.String(load_default='')
+    counterparty = fields.String(load_default=None)
+    address = fields.String(load_default=None)
+
+
+class NewCalendarEntrySchema(CalendarCommonEntrySchema):
+    timestamp = TimestampField(required=True)
+
+
+class DeleteCalendarEntrySchema(IntegerIdentifierSchema):
```

Comment:
You don't need to create a class if nothing new is added. Just use `IntegerIdentifierSchema`